src/internal/index.ts

```diff
@@ -27,4 +27,4 @@ export {
 } from './direction';
 export { useFocusVisible } from './focus-visible';
 export { KeyCode, isModifierKey } from './keycode';
-export { getGlobalFlag } from './global-flags';
+export { getGlobalFlag, setGlobalFlag } from './global-flags';
```

Comment:
Related PR: https://github.com/cloudscape-design/component-toolkit/pull/81

I decided to introduce separate `internal/testing` export for test-only code. Do you think this one could belong there as well?

Reply:
totally. Do you want to bundle the change in that PR, or should I do it later as a follow-up?

Reply:
I will append my with the move. This is more like an FYI where to find this method later

Reply:
Good to know. I will wait for your change before I update this PR: https://github.com/cloudscape-design/components/pull/2478